changelogs/head.asciidoc

```diff
@@ -6,6 +6,8 @@ https://github.com/elastic/apm-server/compare/7.9\...master[View commits]
 [float]
 ==== Breaking Changes
 
+* Renamed `transaction.aggregation.*` config to `transaction.aggregation.transactions.*` {pull}4083[4083]
```

Comment:
If it is not documented/official, I don't think it counts as a breaking change. Like, users reading this will not be able to find what broke

Reply:
Good call - I'll remove this.